docs/internals/contributing/triaging-tickets.txt

```diff
@@ -49,8 +49,8 @@ attribute easily tells us what and who each ticket is waiting on.
 Since a picture is worth a thousand words, let's start there:
 
 .. image:: /internals/_images/triage_process.*
-   :height: 501
-   :width: 400
+   :height: 750
+   :width: 600
```

Comment:
```suggestion
   :width: 475
```
I'm conscious that a mobile screen is usually around the 400 mark and I'm not sure if images are responsive on mobile
This is bigger than the original but not huge - what do you think?

Reply:
the difference is noticeable in desktop, and it helps make it more legible; it doesn't affect mobile responsiveness

Reply:
Yes, the CSS has a `max-width: 100%` to ensure images will resize :) (I’ve tested).